test/MLOps.NET.Tests/EvaluationCycleCatalogTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MLOps.NET.Catalogs;
+using MLOps.NET.Entities.Interfaces;
+using MLOps.NET.Storage;
+using MLOps.NET.Utilities;
+using Moq;
+using System;
+using System.Threading.Tasks;
+
+namespace MLOps.NET.Tests
+{
+    [TestCategory("UnitTests")]
+    [TestClass]
+    public class EvaluationCycleCatalogTests
```

Comment:
I think this has been discussed before but cannnot remember what the decision was. Should this be EvaluationCatalogTests? LifeCycle is a different class.

Reply:
Great catch, that was a typo from copy-pasting the old test class :)